index.html

```diff
@@ -0,0 +1,357 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <script src="https://kit.fontawesome.com/9ac36e1f47.js" crossorigin="anonymous"></script>
+    <link rel="stylesheet" href="style.css">
+    <title>Youtube</title>
+</head>
+<body>
+    <div class="nav">
+        <div class="logo"><img src="media/YouTube-Logo.wine.svg" alt="logo" width="100px"></div>
+        <div class="search-placeholder">
+           <div class="search">
+            <input type="text" placeholder="Search.." name="search">
+            <button type="submit" class="button-submit"><i class="fa-sharp fa-solid fa-magnifying-glass"></i></button>
+            </div>
+            <span><i class="fa-sharp fa-solid fa-microphone"></i></span>
+        </div>
```

Comment:
Please use nav tag for this section...these are good for SEO and can make your code more readable and understandable.